frontend/src/domain/shared_slices/index.ts

```diff
@@ -54,11 +54,9 @@ const homeReducers = {
   vesselList: vesselListReducer
 }
 
-const backofficeReducers = {
```

Comment:
This is not the persisted root reducer, but another specific reducer for backoffice (with less reducers) which is persisted. The naming is a bit confusing I think and do not reflect the usage, I propose to keep the original naming, wdyt ?